src/content/docs/configuration/providers.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Providers
+description: Understand Providers and configure new Targets.
+sidebar:
+  label: Providers
+---
+
+Daytona abstracts the management and deployment of Workspaces into 2 related concepts:
+
+1.  __Provider__  
+    A plugin that interfaces with [Daytona Server](/reference/server), responsible for Workspace deployment and lifecycle management.
+    Examples of platforms Providers may connect with include:
+    *   Container management platforms such as Docker or Podman;
+    *   Cloud hosting providers such as AWS or DigitalOcean.
+2.  __Target__  
+    A set of configuration that governs how Daytona manages and deploys Workspaces.
+    Each Target is tied to an individual Provider.
+    Daytona allows you to configure new and existing Targets, and choose between them at Workspace creation time.
+
+Daytona's architecture decentralizes the role of a Provider into a separate service.
+The [Daytona Server](/reference/server) communicates with Providers to execute operations relating to Workspace creation and lifecycle management.
+
+This makes it possible to create new Providers to govern Workspace management on any platform or hosting providers.
+
+## Managing Providers
+### Install a Provider
+You can install a Provider.
+Once a new Provider is installed, you can use it when setting a Target.
+
+__Prerequisites__
+*   A working installation of Daytona.
+
+__Procedure__
+1.  Execute the following command to install a new Provider:
+    ```shell
+    daytona provider install
+    ```
+2.  Select the Provider you want to install from the list.
+
+### Uninstall a Provider
+You can uninstall an existing Provider.
+
+__Prerequisites__
+*   A working installation of Daytona.
+*   At least 1 Provider installed.
+
+__Procedure__
+1.  Execute the following command to uninstall a Provider:
+    ```shell
+    daytona provider uninstall
+    ```
+2.  Select the Provider you want to uninstall from the list.
+
+### Update a Provider
+You can update an existing Provider.
+This allows you to apply the latest enhancements and bug fixes.
+
+__Prerequisites__
+*   A working installation of Daytona.
+*   At least 1 Provider installed.
+
+__Procedure__
+1.  Execute the following command to update a Provider:
+    ```shell
+    daytona provider update
+    ```
+2.  Select the Provider you want to update from the list.
+
+## Managing Targets
+### Set a Target
+You can set a new Target for Daytona to use when managing Workspaces.
+
+__Prerequisites__
+*   A working installation of Daytona.
+*   At least one Provider installed.
+
+__Procedure__
+1.  Execute the following command:
+    ```shell
+    daytona target set
+    ```
+2.  Select the appropriate Provider for the environment you want to deploy to.
+3.  Select `New Target`.
+4.  Enter a name for your Target.
+5.  Enter the appropriate configuration options when prompted.
+6.  Click `Enter` to add the Target.
+
+### Remove a Target
+You can remove an existing target.
```

Comment:
This one as well.
The second line I suggest you emphasize with a note or something.